src/arch/armv8/armv8-a/aarch64/vmm.c

```diff
@@ -18,7 +18,7 @@ void vmm_arch_init_tcr()
      * TODO: we could make this more dynamic and adapt it to each virtual machine.
      */
 
-    static size_t min_parange = 0b111;
+    static size_t min_parange = 0x7;//0b111;
```

Comment:
We could leave the comments with the bit pattern ou. If we are keeping these comments at least clean them up:

```suggestion
    static size_t min_parange = 0x7; // 0b111
```

Reply:
Fixed in commit a27a15bff8ad78e2cabf764fe795b62650daf18f